src/lesson8/taskNumber3/UserAnimalChoise.java

```diff
@@ -0,0 +1,19 @@
+package lesson8.taskNumber3;
+
+public enum UserAnimalChoise {
+    CAT(0),
+    DOG(1),
+    FISH(2);
+    private final int userChoice;
```

Comment:
Seems requirements have at least five types? Task in this variaty doesn't have sense